docs/contributing/coding-guidelines.md

```diff
@@ -109,10 +92,6 @@ To know if your end-to-end test belongs to the `smoke` folder or the `regression
 
 If you answered "no" to these questions, then the test belongs in the `regression` folder.
 
-### Reduce Module Dependencies
-
-Always try to reduce module dependencies.
-
 ### Layers in Spartacus
 
 `occ -> ngrx/store -> component`
```

Comment:
I believe we should also update the section regarding ngrx, now that facades are ready.

Reply:
GH-1690 has been created as a follow-up ticket to get the doc input for updating the section on NGRX.